### PR TITLE
fix(project-access): handle backslash and leading/trailing slashes urlPath

### PR DESCRIPTION
**Pull request by Klaus-Keller**
## Issue
https://github.com/SAP/open-ux-tools/issues/1098

## Description
Small fix for internal function `findServiceByUri()`. When searching for service by URI we should ignore leading and trailing slashes.

### CODE REVIEW

**Review by devinea — APPROVED** [review]
Code changes, tests and changset looks as expected.

File: packages/project-access/src/project/cap.ts [hunk 1]
```diff
@@ -151,8 +151,9 @@ function findServiceByUri(
     services: { name: string; urlPath: string }[],
     uri: string
 ): { name: string; urlPath: string } | undefined {
-    const searchUri = uri.replace(/[\\/]/g, '/'); // replace all \ and / in service uri due to issues on Windows with backslashes
-    return services.find((srv: { name: string; urlPath: string }) => srv.urlPath.replace(/[\\/]/g, '/') === searchUri);
+    const uniformUri = (uri: string) => uri.replace(/[\\/]/g, '/').replace(/(?:^\/)|(?:\/$)/g, '');
```
Comment (slin-sap) [inline packages/project-access/src/project/cap.ts:154]: IMO, the original comment should be kept and extended to explain the 2nd replace is used to remove leading and trailing slashes.

Comment (Klaus-Keller) [reply packages/project-access/src/project/cap.ts]: changed it and added comment

**Review by devinea — APPROVED** [review]
Thanks @Klaus-Keller 
Changes to code and tests look good. 
Changeset ✅

**Review by slin-sap — APPROVED** [review]
- Understand the proper fix will be required on cds side.
- Fix lgtm
- patch level is justified since there is no API level change.

**Pull request merged**